app-misc/wcd/files/wcd-6.0.4-fix-build-for-clang16.patch

```diff
@@ -0,0 +1,30 @@
+Clang16 will not allow implicit function declarations. 
```

Comment:
Sent upstream?

Reply:
Will do when this is merged and working. I don't want to sent broken patches.

 Besides, upstream has no GitHub, so in can not put a link to a PR here.

Reply:
... and I'm glad I did ;)

So, in this case, they are missing prototypes -- but they're not standard C ones, instead they're from curses!

Take a look at `man mvwaddwstr` :)

Reply:
Yes I noticed that is from curses.

But importing curses.h doesn't help, I think that is because in curses those functions are not normal functions but macros.

Reply:
Try importing ncursesw/curses.h?

Reply:
Let's do this in irc instead ok?

Reply:
ok